.github/workflows/dapr-test.yml

```diff
@@ -148,7 +148,8 @@ jobs:
         if: env.TEST_CLUSTER != ''
         run: |
           make setup-helm-init
-          make setup-test-env
+          make setup-test-env-redis
+          make setup-test-env-kafka
```

Comment:
It is explained in the description: e2e test in AKS does not need mongodb deployment.
This deployment does not work on Windows cluster, which contributes to cluster instability.

Reply:
`setup-test-env` is defined as `setup-test-env: setup-test-env-kafka setup-test-env-redis setup-test-env-mongodb`

So all you are doing is remove `setup-test-env-mongodb`.

Why?

Reply:
This part was not in the description: "This deployment does not work on Windows cluster, which contributes to cluster instability." :)

It makes sense now.

Instead of changing this, can we change the Makefile to do a no-op for setup-test-env-mongodb when OS is Windows? I think it will be more sustainable over time to maintain that as we add more component to E2E tests.

Reply:
mondoDB is not required neither for Linux nor for Window in E2E-AKS.

Reply:
I am OK with this change for now but we need a better solution to select the components upfront and have the target in makefile to respect that list.